### PR TITLE
Support multiple project ID for run folder transfer

### PR DESCRIPTION
**Pull request by chuan-wang**

### CODE REVIEW

File: taca/analysis/analysis.py [hunk 1]
```diff
@@ -155,17 +155,18 @@ def transfer_runfolder(run_dir, pid, exclude_lane):
...
 
     :param: string run_dir: the run to transfer
-    :param: string pid: the project to include in the SampleSheet
+    :param: string pid: the projects to include in the SampleSheet separated by comma
     :param: string exclude_lane: lanes to exclude separated by comma
 
     """
     original_sample_sheet = os.path.join(run_dir, 'SampleSheet.csv')
-    new_sample_sheet = os.path.join(run_dir, pid + '_SampleSheet.txt')
+    pid_list = list(set(pid.strip().split(',')))
```
Comment (aanil) [inline taca/analysis/analysis.py:163]: One more strip after the split maybe?

File: taca/analysis/analysis.py [hunk 2]
```diff
@@ -246,14 +247,14 @@ def transfer_runfolder(run_dir, pid, exclude_lane):
...
-def extract_project_samplesheet(sample_sheet, pid):
+def extract_project_samplesheet(sample_sheet, pid_list):
     header_line = ''
     project_entries = ''
     with open(sample_sheet) as f:
         for line in f:
             if line.split(',')[0] in ('Lane', 'FCID'):  # include the header
                 header_line += line
-            elif pid in line:
+            elif any(pid in line for pid in pid_list):
```
Comment (aanil) [inline taca/analysis/analysis.py:257]: How many pids will be in the list usually?

File: taca/analysis/analysis.py (hunk 1 above)
Comment (aanil) [reply taca/analysis/analysis.py:163]:
```suggestion
    pid_list = [x.strip() for x in pid.split(',')]
```
maybe?

File: taca/analysis/analysis.py (hunk 2 above)
Comment (chuan-wang) [reply taca/analysis/analysis.py:257]: Between 2 and 10 I guess.

Comment (aanil) [reply taca/analysis/analysis.py:257]: Should be fine, I hope.

**Review by aanil — APPROVED** [review]
👍

**Pull request merged**